### ISSUE TITLE
http://www.tdzk.com/ unresponsive

### ISSUE THREAD

**Issue by paulderaadt** [zmaniacz/shipcalc#1]
Expected behavior:
Visit http://www.tdzk.com/ 
Login
Create big memories
Actual behavior:
We sit in discord and talk about politics and cooking

**Comment by ketukil** [comment]
Yes, I feel the same. Days from exploring Pleiora Nebula and resourcing asteroid fields of Zane in are long gone.
